src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(num) {
```

Comment:
Consider adding a brief comment above the adder function to explain its behavior. While the code is clear to those familiar with the pattern, a short description can be helpful for maintenance and readability.